falcon/vendor/mimeparse/mimeparse.py

```diff
@@ -56,7 +63,7 @@ def parse_media_range(range):
     :rtype: (str,str,dict)
     """
     (type, subtype, params) = parse_mime_type(range)
-    params.setdefault('q', params.pop('Q', None))  # q is case insensitive
+    params.setdefault('q', params.pop('Q', ''))  # q is case insensitive
```

Comment:
not sure if this is equivalent to None

Reply:
cc @vytas7

Reply:
We should IMHO undo annotations to this module altogether, let alone changes that might affect functionality. It is vendored, so we should keep it verbatim.
(I'm planning to reimplement the subset we are actually using and stop vendoring, but until then we should keep it _as is_.)

Reply:
In this case it is because two lines below it's set to '1' either if it is None or ''.

Reply:
Removed typing from vendor and excluded it from typing.